src/components/Page/Page.tsx

```diff
@@ -166,7 +166,9 @@ class PageInner extends React.PureComponent<ComposedProps, never> {
     return {
       title,
       buttons: transformActions(appBridge, {
-        primaryAction,
+        primaryAction: isPrimaryAction(primaryAction)
```

Comment:
@BPScott Thoughts on how I'm handing the typing here & the helper function in general? Will probably ping you for review too, but was curious on your initial thoughts.

Reply:
This seems alright at first glance. AppBridge integration is going away in v5. The PR has been open for an age and hopefully we'll get around to releasing it in 2 months, so I'd prioritize how to handle this stulf in non-AppBridge-land which you're doing anyway.

Reply:
There's a fancy way to add optional object params

```tsx
{
        ...(isPrimaryAction(primaryAction) && {primaryAction})
}
```